Makefile

```diff
@@ -1,18 +1,22 @@
 
-push: push-8.0 push-7.2 push-7.1 push-7.0
+push: push-8.1 push-8.0 push-7.2 push-7.1 push-7.0
+
+push-8.1: build-8.1
+	docker tag jbfavre/vertica:8.1.0-1_ubuntu-14.04 jbfavre/vertica:latest
```

Comment:
I'm not that confortable enough to use Vertica 8.1 as latest.
Indeed, it's the most recent release, but it's as released only 2 days ago.

Though, I agree to switch from 7.2 to 8.0